content/en/monitoring/notification-manager.md

```diff
@@ -405,4 +407,4 @@ spec:
 EOF
 ```
 
->Global slack receiver use the default slack config.
+> Global slack receiver use the default slack config.
```

Comment:
Global slack receiver use the default slack config. =>
Global slack receiver will use the default slack config.